src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.DecryptToken.cs

```diff
@@ -14,6 +15,39 @@ namespace Microsoft.IdentityModel.JsonWebTokens
 {
     public partial class JsonWebTokenHandler : TokenHandler
     {
+        internal async Task<ValidationResult<string>> DecryptTokenWithConfigurationAsync(
```

Comment:
- Don't we need a cancellation token?
- Why is the method taking ValidationParameters internal?

Reply:
This is the unreleased APIs (the new validation model refactorings). I added a method here for consistency, so it doesn't get out of sync.

Reply:
I thought this is the one we use in "AuthN" ?

Reply:
At least right now, we still use the current IdentityModel API, not this new one.